### DISCUSSION TITLE
Crashes Toolbar Buttons and Right Click Menu.

### DISCUSSION THREAD

**Discussion by soul4kills** [QL-Win/QuickLook#1208] [General]
When using quicklook, after a while, right click doesn't work anymore and the explorer toolbar buttons don't work either including tabs.

I'm using Windows 11 insider build 22H2 25236.1010.

At first I thought it was just the insider build but I turned off quicklook and it stopped happening. Then tried quicklook again and it started happening again.

**Comment by xupefei** [comment]
What crashes? Explorer or QuickLook?

**Reply by soul4kills** [reply]
Sorry, crashing is the wrong terminology. Don't really know the word to describe what's happening. Quicklook continues to work without disruption, just explorer stops responding.

Explorer's UI elements stop working. It's possible it's because of the new tabbed browsing. Tested explorer for a while without quicklook on and hasn't happened. Then tried using quicklook and it happens again.

What happens is right click on files stops working and the ui elements like toolbar buttons and tabs stop responding. To fix, I would have to close explorer and start it again.

Creating this discussion to see if anyone else is experiencing the same problem using quicklook. I googled for my issue but doesn't seem like other Insider users are having the same issues.

**Reply by soul4kills** [reply]
Disregard, I figured it out. It was because of icloud for windows. Uninstalled it and all is fine.
